catinabox/services/food_truck_service.py

```diff
@@ -0,0 +1,11 @@
+from .util import http
```

Comment:
Was going to bring it over to the cattery too. It's just some magic to automatically `json.dumps` the return of an endpoint.

e.g.
```python
return 200, {'foo': 'bar'} ====> return 200, json.dumps({'foo': 'bar'})
```